staging/src/k8s.io/kubectl/docs/book/pages/examples/kustomize.md

```diff
@@ -270,7 +270,7 @@ vars:
 # specified by kustomize's configuration data.
 #
 # The default config data for vars is at
-# https://github.com/kubernetes-sigs/kustomize/blob/master/pkg/transformers/config/defaultconfig/varreference.go
+# https://github.com/kubernetes-sigs/kustomize/blob/master/api/konfig/builtinpluginconsts/varreference.go
```

Comment:
can you point to a stable tag/branch?

Reply:
https://github.com/kubernetes-sigs/kustomize/blob/v3.3.1/pkg/transformers/config/defaultconfig/varreference.go?Ok?

Reply:
@liggitt